specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/CreateOrUpdateWorkspace.json

```diff
@@ -3,7 +3,7 @@
     "subscriptionId": "00000000-1111-2222-3333-444444444444",
     "resourceGroupName": "resourceGroup1",
     "workspaceName": "workspace1",
-    "api-version": "2019-06-01-preview",
+    "api-version": "2020-04-01-preview",
```

Comment:
I presume the changes in this folder 2019-06-01-preview are not intentional? Please revert.

Reply:
Updated the changes.